metadata/mcf/msc_nwp_riopsf.yml

```diff
@@ -0,0 +1,65 @@
+
+base_mcf: /share/contact-main.yml
+
+mcf:
+    version: 1.0
+
+metadata:
+    charset: utf8
+    hierarchylevel: dataset
+    identifier: urn:wmo:md:${MET5_CENTRE_ID}:ocean:medium-range:nwp.riops
```

Comment:
`s/MET5_CENTRE_ID/WIS2_CENTRE_ID/g`

Reply:
Intentional change, didn't think we'd reuse the exact same ones, but I can be mistaken.

Reply:
We are using the identical centre id as WIS2 proper, yes.

Reply:
Ok, noticed your github action, adjusting.